src/component/ProductDetail.js

```diff
@@ -1,125 +1,34 @@
-                  </h4>
-                  <p>See more snippets like this online store item at <a target="_blank" href="http://www.bootsnipp.com">Bootsnipp - http://bootsnipp.com</a>.</p>
-              </div>
-              <div className="ratings">
-                  <p className="pull-right">15 reviews</p>
-                  <p>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                  </p>
-              </div>
-          </div>
-      </div>
+import React from "react";
+import Rating from "./Rating";
 
-      <div className="col-sm-4 col-lg-4 col-md-4">
-          <div className="thumbnail">
-              <img src="http://placehold.it/320x150" alt=""/>
-              <div className="caption">
-                  <h4 className="pull-right">$64.99</h4>
-                  <h4><a href="#">Second Product</a>
-                  </h4>
-                  <p>This is a short description. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-              </div>
-              <div className="ratings">
-                  <p className="pull-right">12 reviews</p>
-                  <p>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star-empty"></span>
-                  </p>
-              </div>
-          </div>
-      </div>
+function ProductDetail(props) {
 
-      <div className="col-sm-4 col-lg-4 col-md-4">
-          <div className="thumbnail">
-              <img src="http://placehold.it/320x150" alt=""/>
-              <div className="caption">
-                  <h4 className="pull-right">$74.99</h4>
-                  <h4><a href="#">Third Product</a>
-                  </h4>
-                  <p>This is a short description. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-              </div>
-              <div className="ratings">
-                  <p className="pull-right">31 reviews</p>
-                  <p>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star"></span>
-                      <span className="glyphicon glyphicon-star-empty"></span>
-                  </p>
-              </div>
+  const productDivs = props.product.map(product => {
```

Comment:
Slight misunderstanding here. The ProductDetail component is suppose to represent one product.  It is the reusable representation of one thing.
App.js is the one who is suppose to do the mapping. What if there was another component that just wanted to show one product, like the product of the day or something.  It should be able to use ProductDetail also.